index.html

```diff
@@ -3,47 +3,229 @@
+      {"state":"Maryland","region":"South","days":24.8,"reserves":"$1,086 "},
+      {"state":"Massachusetts","region":"Northeast","days":14.2,"reserves":"$1,485 "},
+      {"state":"Michigan","region":"Midwest","days":26,"reserves":"$668 "},
+      {"state":"Minnesota","region":"Midwest","days":25.4,"reserves":"$1,421 "},
+      {"state":"Mississippi","region":"South","days":30.5,"reserves":"$461 "},
+      {"state":"Missouri","region":"Midwest","days":22.9,"reserves":"$548 "},
+      {"state":"Montana","region":"West","days":76.6,"reserves":"$455 "},
+      {"state":"Nebraska","region":"Midwest","days":132.2,"reserves":"$1,460 "},
+      {"state":"Nevada","region":"West","days":15.7,"reserves":"$146 "},
+      {"state":"New Hampshire","region":"Northeast","days":24.1,"reserves":"$83 "},
+      {"state":"New Jersey","region":"Northeast","days":7,"reserves":"$627 "},
+      {"state":"New Mexico","region":"West","days":36.7,"reserves":"$634 "},
+      {"state":"New York","region":"Northeast","days":42.4,"reserves":"$7,300 "},
+      {"state":"North Carolina","region":"South","days":25.7,"reserves":"$1,516 "},
+      {"state":"North Dakota","region":"Midwest","days":147.2,"reserves":"$1,303 "},
+      {"state":"Ohio","region":"Midwest","days":37,"reserves":"$3,190 "},
+      {"state":"Oklahoma","region":"South","days":24.7,"reserves":"$434 "},
+      {"state":"Oregon","region":"West","days":38.5,"reserves":"$868 "},
+      {"state":"Pennsylvania","region":"Northeast","days":0.1,"reserves":"$9 "},
+      {"state":"Rhode Island","region":"Northeast","days":37.1,"reserves":"$351 "},
+      {"state":"South Carolina","region":"South","days":63.3,"reserves":"$1,182 "},
+      {"state":"South Dakota","region":"Midwest","days":45,"reserves":"$171 "},
+      {"state":"Tennessee","region":"South","days":38.3,"reserves":"$1,311 "},
+      {"state":"Texas","region":"South","days":119.4,"reserves":"$15,838 "},
+      {"state":"Utah","region":"West","days":56.8,"reserves":"$895 "},
+      {"state":"Vermont","region":"Northeast","days":19.4,"reserves":"$76 "},
+      {"state":"Virginia","region":"South","days":14.3,"reserves":"$715 "},
+      {"state":"Washington","region":"West","days":30.1,"reserves":"$1,379 "},
+      {"state":"West Virginia","region":"South","days":111.1,"reserves":"$1,289 "},
+      {"state":"Wisconsin","region":"Midwest","days":9.8,"reserves":"$416 "},
+      {"state":"Wyoming","region":"West","days":197.5,"reserves":"$960 "}
+    ];
+    var MAX_DAYS = 447
+    var options = {
+      sorted: false,
+      filtered: false,
+      page: 0
+    };
+  
+    function updateTable() {
+      var data = states.slice();
+      if (options.sorted) {
+       if (options.sorted === 'asc') {
+          data.sort(function (a, b) { return a.days - b.days; });
+        } else if (options.sorted === 'desc') {
+          data.sort(function (a, b) { return b.days - a.days; });
+        }
+      }
+      var table = document.getElementById('data-table');
```

Comment:
I don't know if you attempted to write the filter logic and just deleted it, but this is where it would go, before we start populating the table. Hopefully our progress together in class (which you can find [here](https://github.com/JHU-DataViz-Summer16/week-2-js-dom/blob/master/index.html#L232)) is helpful for getting started, but ping Alex and me with any questions/problems.